test/Conversion/tritongpu_to_llvm_sm120.mlir

```diff
@@ -0,0 +1,32 @@
+// RUN: triton-opt %s -split-input-file --convert-triton-gpu-to-llvm=compute-capability=120 -cse | FileCheck %s
```

Comment:
Are there any differences for sm_120 in codegen? We could skip those maybe or you ca add this command line in the other file so that we don't need to duplicate the tests? (or run both command line in this file)

Reply:
Moved this to tritongpu_to_llvm.mlir